src/function/scalar/math/numeric.cpp

```diff
@@ -206,6 +206,21 @@ struct BitCntOperator {
 	}
 };
 
+struct HugeIntBitCntOperator {
+	template <class TA, class TR>
+	static inline TR Operation(TA input) {
+		using TU = typename std::make_unsigned<int64_t>::type;
```

Comment:
`decltype(input.upper)`!

Reply:
I'm not quite clear what you are suggesting here - can you clarify?  Thank you.  (I'm sorry - I'm not an experienced C++ developer) - I'm selfishly just trying to get hugeint support for bit_count for some bitmap experiments I'm doing... )

Reply:
No worries - I was tired and making an "excessive C++" joke. It was just a comment, not part of the review.  Sorry for the confusion.

Reply:
Cool, thanks!